lib/bindings/python/src/dynamo/health_check.py

```
@@ -15,7 +15,16 @@
 
 logger = logging.getLogger(__name__)
 
-__all__ = ["HealthCheckPayload", "load_health_check_from_env"]
+__all__ = [
+    "HealthCheckPayload",
+    "load_health_check_from_env",
+    "is_health_check_enabled",
+]
```

Ruff (0.15.10)

[warning] `__all__` is not sorted

Apply an isort-style sorting to `__all__`

(RUF022)